.github/workflows/ci.yml

```diff
@@ -34,7 +34,18 @@ jobs:
     name: Arduino CLI Build
     steps:
       - uses: actions/checkout@v2
-      - name: Test
+
+      - name: Apply serial patch on tflite-micro
+        run: |
+          sed --in-place '/#define DEBUG_SERIAL_OBJECT/s/(Serial)/(Serial1)/' src/tensorflow/lite/micro/system_setup.cpp
```

Comment:
I still don't really like it. I believe we have a problem that we do not compile the USB code at all now.

Could we:
- checkout
- use the original `Test` step
- apply the patch
- rebuild, cleanly
- test in Renode

This will prolong the test, but we can try to argue for it (and we're trying to get rid of this hack anyway)

Reply:
I modified it, please take a look if that's what you meant